Practice/kruglov/home_task_4/PyLec4T5.py

```diff
@@ -0,0 +1,18 @@
+# 5 Interpolation
+
+
+def inter(str1, dict1):
+    str2 = ""
+    for el in str1.split():
+        if el in dict1:
+            str2 += dict1[el]
+        else:
+            str2 += el + " "
```

Comment:
It's better to collect elements to some list (not to a string) and then just call " ".join(res_list) to avoid any troubles with spaces between symbols and in the end of the string. Because currently we may get incorrect result for this example: "sss is ancient city" -> "Spartais ancient city ".

Reply:
Thanks for the tip) Updated as recommended.